### PR TITLE
Update : Make page obey HTML5 rules

### PR DESCRIPTION
**Pull request by smartclash**
Lol, read the title

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this issue

File: index.html [hunk 1]
```diff
@@ -1,112 +1,117 @@
...
     <link rel="stylesheet" href="tacit.min.css"/>
     <title>tacit</title>
     <link rel="shortcut icon" type="image/gif" href="logo.gif"/>
-  </head>
-  <body>
-    <section>
-      <header>
+</head>
+<body>
+<section>
```
Comment (yegor256) [inline index.html]: @smartclash I can't understand the logic. Why do you indent `header` but don't indent `section` and `body`? :)

**Comment by yegor256** [comment]
@smartclash see my comment above. Also, what's wrong with two-spaces indentation? Four spaces will just make file less readable. Don't you think?

File: index.html (hunk 1 above)
Comment (smartclash) [inline index.html]: Well, I am really sorry. It was a code cleanup done by my IDE. So, the project got indented by four spaces. And as you said, I don't know how `section` and `body` escaped from my hands @yegor256

**Comment by smartclash** [comment]
DONE!! DOUBLE CHECKED @yegor256

**Comment by yegor256** [comment]
@smartclash there is still something wrong with the indentation, see: https://github.com/yegor256/tacit/pull/66/files You shifted `head` and `body` left :) Don't do that. Every HTML element must be shifted right by exactly two spaces. No matter what level it is on. Please, fix and I'll merge. Thanks :)

**Comment by filfreire** [comment]
@smartclash can you solve the conflicts in `index.html`?
Thanks 👍

**Comment by filfreire** [comment]
@yegor256 the user @smartclash hasn't solved the conflicts since May and after I mentioned him 17 days ago. I believe we can close this PR.

@smartclash if you wish to re-open the PR I suggest fixing the conflicts and opening a new one.
This issue was of "Making the page obey HTML5 rules" was also done in recent commits by other users.

**Comment by filfreire** [comment]
@yegor256 I believe we can close this PR.

@smartclash again, if you wish to re-open the PR I suggest fixing the conflicts and opening a new one.
This issue was of "Making the page obey HTML5 rules" was also done in recent commits by other users. Still, thank you for your contributions 👍

**Comment by yegor256** [comment]
@filfreire closed

**Pull request closed without merging**